### ISSUE TITLE
julec: compile issues on Windows

### ISSUE THREAD

**Issue by JigolKa** [the-xlang/x#33]
### Description

Hello,
I tried to compile a Hello World program under Windows and got an error about some numbers. Here's the log:
```
In file included from C:\jule\api\derive/clone.hpp:8,
                 from C:\jule\api\derive/derive.hpp:8,
                 from C:\jule\api\jule.hpp:8,
                 from C:\Users\Noam\dev\test\dist\ir.cpp:8:
c:\jule\api\types.hpp:34:54: error: exponent has no digits
   34 |     constexpr jule::F32 MAX_F32{ 0x1p127 * (1 + (1 - 0x1p-23)) };
      |                                                      ^~~~
c:\jule\api\types.hpp:35:55: error: exponent has no digits
   35 |     constexpr jule::F32 MIN_F32{ -0x1p127 * (1 + (1 - 0x1p-23)) };
      |                                                       ^~~~
c:\jule\api\types.hpp:36:55: error: exponent has no digits
   36 |     constexpr jule::F64 MAX_F64{ 0x1p1023 * (1 + (1 - 0x1p-52)) };
      |                                                       ^~~~
c:\jule\api\types.hpp:37:56: error: exponent has no digits
   37 |     constexpr jule::F64 MIN_F64{ -0x1p1023 * (1 + (1 - 0x1p-52)) };
      |                                                        ^~~~
cc1plus.exe: warning: unrecognized command line option '-Wno-everything'
external compiler failed with status code: 1
```

I think the problem comes from `0x1p`: it's an invalid hex number.

### Expected behavior

The code should have compiled successfully

### Current behavior

Compiler returns exit code 1

### Additional information

I tried the same code under Ubuntu, works fine.

I use gcc version `9.2.0`
I'm on Windows 10

**Comment by mertcandav** [comment]
Hi,

Thanks for your time and effort.
This is the known issue and fixed in latest source code. Probably you are using releases but releases has some bugs like this. Therefore I recommend compile JuleC from IR to experience Jule. [Here is the IR repository of JuleC](https://github.com/julelang/julec-ir).

Feel free join and ask any question at our Discord Community server, issues or discussions.

**Comment by mertcandav** [comment]
This issue fixed with Beta 0.0.3 release.
